Charts/Classes/Renderers/CandleStickChartRenderer.swift

```diff
@@ -43,7 +43,7 @@ public class CandleStickChartRenderer: LineScatterCandleRadarChartRenderer
         }
     }
     
-    private var _shadowPoints = [CGPoint](repeating: CGPoint(), count: 4)
+    private var _shadowPoints = [CGPoint](repeating: CGPoint(), count: 2)
```

Comment:
hmm, it seems get complicated... I mean the conversion tool will not change count 4 to 2 right? It seems like a manual change. I am asking why we change from 4 to 2.
Also, why you said "many changes in this beta were not suggested" ? I am still downloading, but I need to work then.